### ISSUE TITLE
Blocks Flashing When Used With Sodium (1.18.2)

### ISSUE THREAD

**Issue by taniskint** [maruohon/minihud#223]
Hi! So the issue is pretty simple: When MiniHUD and Sodium are installed on my wife's computer, blocks flash every time there's a block update. This is most obvious when breaking/placing blocks, but it also happens when i.e. redstone triggers. All blocks that aren't full blocks flash for a frame or two (path blocks, stairs, slabs, glass panes, and the like).

When using just MiniHUD (and its dependencies), no flashing occurs. When using just Sodium, no flashing occurs, so this seems to be an interaction between the two mods. She has no other mods installed.

Let me know if you need any other information.

### Mod information
(at the time of writing, these are all the latest versions)
- Sodium version: 0.4.1 build 15
- MiniHUD version: 0.22.0
- Malilib version: 0.12.0
- Fabric API version: 0.47.10

### Computer info
It's a Surface Pro 7, so
- Intel i5-1035G4 (integrated graphics)
- 8GB RAM

**Comment by maruohon** [comment]
Is MiniHUD simply being installed (and/or the main rendering (`H`) being enabled) enough, or do you have some overlay renderers enabled, such as some shapes created and enabled or the slime chunk overlay or the structure bounding box overlay etc. enabled?

**Comment by taniskint** [comment]
Simply it being installed is enough. We tried disabling it with `H`, and it still flashed. We checked and there are no renderers enabled.

**Comment by maruohon** [comment]
No idea what would be causing that if it even happens with MiniHUD rendering disabled. Do you get any errors or warning in the game console/log that might be related? Those would likely happen when breaking/placing blocks.

I haven't heard of a similar issue before, and this also doesn't happen in my test instance, so I don't really have any way to reproduce or debug this on my end. It might be related to the slower(?) CPU/GPU in that system.

One thing you can try (that is somewhat unlikely to make a difference though), is to take out the Fabric API mod, if you don't use other mods that require it. It's not required separately for any of my mods (or Sodium, at least as of now).

**Comment by taniskint** [comment]
Yeah, I agree it's strange. No warnings or anything (that I've seen). It just started out of nowhere, too. I submitted an issue with Sodium as well, so we'll see if anything comes of that.
